### ISSUE TITLE
NODE_TLS_REJECT_UNAUTHORIZED warning

### ISSUE THREAD

**Issue by kellymears** [roots/bud#23]
* [x] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
* [x] This request isn't a duplicate of an existing issue
* [x] I've read the [docs](https://roots.io/documentation/) and followed them (if applicable)
* [x] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

## Description

`bud init` does a quick series of tasks to determine the plugins URI without prompting the user for it, if the project is WordPress based. This is obviously important given that some users are going to be running their plugins out of `wp-content`, while users of Bedrock and several other tools tend to use `app`. Bud needs this information to template things like `webpack` scripts.

The general flow is:

- check for `wp-config.php` and `index.php` in the expected location
- swap in a dummy `index.php` file for the real one. the dummy returns a JSON object representing the web path to the plugins dir.
- make a `node-fetch` request for the site URL.

The issue is that a lot of local dev environments are not running a proper SSL certificate and so the request fails. Currently, Bud gets around it by setting `NODE_TLS_REJECT_UNAUTHORIZED` to 0 within the context of this call.

https://github.com/roots/bud/blob/f313e8a14a94f359bea51a77674f33f12201d002/src/budfiles/init/init.bud.js#L97-L115

This works but results in the following warning being appended to Bud's output in `stdout`:

```
Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
```

It doesn't seem to matter if `NODE_NO_WARNINGS` is enabled when it comes to suppressing this output. 

## Steps to Reproduce

- run `bud init`.

## What to do

Ideally we wouldn't need to disable low-level security features in the Node runtime, but I also don't want to field a constant stream of questions about how to supply Bud with a local SSL certificate (also, at that point, it would obviously be much easier for them to just type out their project path). 

It also seems like not a big deal to me given that:

- `bud` is a development tool and not at all designed for environments where this sort of verification is important.
- the feature is only disabled in this one context for the purposes of this single call.
- this is a one-and-done command.

Basically, I could be persuaded otherwise but right now I'm not seeing a massive issue with this tiny hack.

But, one way or another, this warning has got to go. Here are the options, as I see them:

- Find another process.env flag that properly disables this warning.
- Run this in a child process so that its output doesn't pollute stdout.
- Find a reliable way to make this request without modifying `NODE_TLS_REJECT_UNAUTHORIZED`.
- Remove this feature (which would require the user to specify their path from web root for the project).

Open to discussion, ideas, PRs.
